webapp/src/modules/wallet/sagas.ts

```diff
@@ -156,18 +156,30 @@ function* handleWallet(
       network: contract.network
     })
 
-    authorizations.push({
-      address,
-      authorizedAddress: marketplace.address,
-      contractAddress: contract.address,
-      contractName:
-        contract.category === NFTCategory.WEARABLE &&
-        contract.network === Network.MATIC
-          ? ContractName.ERC721CollectionV2
-          : ContractName.ERC721,
-      chainId: contract.chainId,
-      type: AuthorizationType.APPROVAL
-    })
+    if (contract.category === NFTCategory.WEARABLE) {
+      authorizations.push({
+        address,
+        authorizedAddress: marketplace.address,
+        contractAddress: contract.address,
+        contractName:
+          contract.network === Network.MATIC
+            ? ContractName.ERC721CollectionV2
+            : ContractName.ERC721,
+        chainId: contract.chainId,
+        type: AuthorizationType.APPROVAL
+      })
+    }
+
+    if (contract.category === NFTCategory.EMOTE) {
```

Comment:
Quick question! Can't we reduce these 2 ifs to just one? 
```suggestion
if (contract.category === NFTCategory.WEARABLE || contract.category === NFTCategory.EMOTE) {

```

Reply:
sure!